wsmaster/codenvy-hosted-system-ram/src/main/java/com/codenvy/service/systemram/SystemRamInfo.java

```diff
@@ -0,0 +1,82 @@
+/*
+ *  [2012] - [2016] Codenvy, S.A.
+ *  All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Codenvy S.A. and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Codenvy S.A.
+ * and its suppliers and may be covered by U.S. and Foreign Patents,
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+package com.codenvy.service.systemram;
+
+import java.util.Objects;
+
+/**
+ * Describes system RAM values and properties.
+ *
+ * @author Igor Vinokur
+ */
+public class SystemRamInfo {
+
+    private final long    systemRamUsed;
+    private final long    systemRamTotal;
+    private final boolean isSystemRamLimitExceeded;
+
+    public SystemRamInfo(long systemRamUsed, long systemRamTotal, boolean isSystemRamLimitExceeded) {
+        this.systemRamUsed = systemRamUsed;
+        this.systemRamTotal = systemRamTotal;
+        this.isSystemRamLimitExceeded = isSystemRamLimitExceeded;
+    }
+
+    /**
+     * Total system RAM amount in Bytes.
+     */
+    public long getSystemRamTotal() {
+        return systemRamTotal;
+    }
+
+    /**
+     * Used system RAM amount in Bytes.
+     */
+    public long getSystemRamUsed() {
+        return systemRamUsed;
+    }
+
+    /**
+     * Returns {@code true} if there is less then 10% of free RAM is present in the system, otherwise returns {@code false}.
```

Comment:
You can't ensure that `it returns true if there is less then 10% of free RAM` because you get this value in constructor.
I'd move logic of init this value in constructor or rewrite docs without description about condition of limit exceeding